assets/js/script.js

```diff
@@ -1,20 +1,108 @@
+const foodSelectorEl = document.querySelector('#foodSelector');
+const randomPairingEl = document.querySelector('#randomPairing');
 
+// hardcoded examples
+let info = "I am a full bodied wine made with grapes. Has been fermenting since the days of old";
+let rating = '⭐️⭐️⭐️⭐️⭐️';
 
+//create a function to remove all html elements except the footer
+const removeExistingElems = () => {
+  headerEl.innerHTML = '';
+	mainEl.innerHTML = '';
+};
 
- // Initializing the DOM Form functions for Materialize (DROP DOWNS)
- // THERE IS A VARIABLE BUG WITH THE VANILLA JS, JQUERY ADDED
-document.addEventListener('DOMContentLoaded', function() {
-	var selectElems = document.querySelectorAll('select');
-	var selectInstances = M.FormSelect.init(selectElems, options);
-	var collapseElems = document.querySelectorAll('.collapsible');
-    var collapseInstances = M.Collapsible.init(collapseElems, options);
-});
+const listenerHandler = el => {
+	el.addEventListener('change', function() {
+		var title = this.options[this.selectedIndex].text;
+		// fetch function dependent on `el`
+		removeExistingElems();
+		// these will go inside the fetch function because that's where we'll receive input for ratings, descr, and recipes
+		infoCardGenerator(title,info, rating); 
+		recipeCardGenerator();
+	});
+};
+
+// refactoring information display function
+const infoCardGenerator = function(title, info, rating) {
+	const infoCardDiv = document.createElement('div');
+	infoCardDiv.setAttribute("id", "content");
+	infoCardDiv.innerHTML = 
+	`<div class="container">
+		<div class="row" id="contentRow">
+			<div class="col s12 m5">
+				<div class="row">
+					<div class="col s12">
+						<div class="card">
+							<div class="card-image">
+									<img src="./assets/images/sample-1.jpg">
+									<span class="card-title">${title}</span>
+							</div>
+							<div class="card-content">
+									<div class ="row">
+											<div class="col s6">
+													<h4>Description</h4>
+													<p>${info}</p>
+											</div>
+											<div class="col s6">
+													<h4>Rating</h4>
+													<p>${rating}</p>
+											</div>
+									</div>
+							</div>
+							<div class="card-action">
+									<a href="#">This is a link</a>
+							</div>
+						</div>
+					</div>
+				</div>
+			</div>
+	</div>`;
+	mainEl.appendChild(infoCardDiv);
 
-// Initializing the DOM Form functions for Materialize (JQuery)
-$(document).ready(function(){
-	$('select').formSelect();
-	$('.collapsible').collapsible()
+	// Call the collapsible function again when the element is rendered
+	$(document).ready(function(){
+	$('.collapsible').collapsible();
 });
-        
\ No newline at end of file
+};
+
+const recipeCardGenerator = (recipe1, recipe2, recipe3) => {
+	const recipeCardDiv = document.createElement("div");
+	const recipeUl = document.createElement("ul");
+	recipeUl.setAttribute("class", "collapsible");
+	recipeCardDiv.classList.add("col", "s12", "m7");
```

Comment:
An alternative way to write this is

`recipeCardDiv.classList = 'col s12 m7';`